packages/components/buttons/icon-button/src/icon-button.figma.tsx

```diff
@@ -0,0 +1,41 @@
+// @ts-nocheck
+import { IconButton } from '@commercetools-frontend/ui-kit';
+import figma from '@figma/code-connect';
+
+/**
+ * -- This file was auto-generated by Code Connect --
+ * None of your props could be automatically mapped to Figma properties.
+ * You should update the `props` object to include a mapping from your
+ * code props to Figma properties, and update the `example` function to
+ * return the code example you'd like to see in Figma
+ */
+
+figma.connect(
+  IconButton,
+  'https://www.figma.com/design/UoqmtHSHwxvEZRcbsM4A3Z/CT-product-design-system?node-id=17%3A15986',
+  {
+    props: {
+      // TODO: Add props here
+      // No matching props could be found for these Figma properties:
+      // "icon": figma.instance('Icon'),
+      // "state": figma.enum('State', {
+      //   "Default": "default",
+      //   "Hover": "hover",
+      //   "Toggled": "toggled",
+      //   "Disabled": "disabled"
+      // }),
+      // "size": figma.enum('Size', {
+      //   "40": "40",
+      //   "30": "30",
+      //   "20": "20",
+      //   "10": "10"
+      // }),
+      // "theme": figma.enum('Theme', {
+      //   "Default": "default",
+      //   "Primary": "primary",
+      //   "Inforimary": "inforimary"
+      // })
+    },
+    example: (props) => <IconButton />,
+  }
+);
```

Comment:
This file was created by the code connect CLI tool but has already been manually updated: 
The `@ts-nocheck` directive was added
The component import statement was fixed
The url on line 15 was manually supplied by us (pasted into the CLI prompt)
The prop mapping is somewhat usable but also mostly incorrect (and what is 'inforimary'?)

Reply:
the more that I think about it, the more that I think that code connect without props is 
a) still pretty helpful
b) a lot more maintainable, since we don't have to update props if/when they change

I wonder if there's a decent way to get the figma asset urls in a batch, and map them to components w/o props?

Reply:
I agree, even knowing which components we're dealing with when doing a handoff would be helpful. 
There *IS* a way, but with what I know right now, it's very convoluted (from the figma community forum: `Select the Variant in the Component Set, open the console in Figma, enter figma.currentPage.selection[0].key.` From there you can request that key from an endpoint and get the node-id). Going to spend a bit more time on that and see what I find.

Reply:
[6644d36](https://github.com/commercetools/ui-kit/pull/3019/commits/6644d36a38cd1503d94c7c6ee020aecf23e4b171) adds a "way" to get the figma asset ids and map them to their matching components. The link.figma.tsx file (below) was created from the updated script, which was able to map ids for 22 components. Of those 22, I think 20 or so are probably necessary (ie a component that we actually want to map and not just a small piece of something that is internal to it's parent component and never used elsewhere).